apps/web/src/app/(dashboard)/(management)/secretariaat/locaties/[id]/_components/import-kwalificaties-dialog.tsx

```
@@ -0,0 +1,390 @@
+  isOpen,
+  setIsOpen,
+}: {
+  locationId: string;
+  isOpen: boolean;
+  setIsOpen: (value: boolean) => void;
+}) {
+  const forceRerenderId = useRef(0);
+
+  return (
+    <ImportKwalificatiesDialogInner
+      key={String(forceRerenderId.current)}
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

/home/jailuser/git/apps/web/src/app/(dashboard)/(management)/secretariaat/locaties/[id]/_components/import-kwalificaties-dialog.tsx:80:19
  78 |   return (
  79 |     <ImportKwalificatiesDialogInner
> 80 |       key={String(forceRerenderId.current)}
     |                   ^^^^^^^^^^^^^^^^^^^^^^^ Passing a ref to a function may read its value during render
  81 |       locationId={locationId}
  82 |       isOpen={isOpen}
  83 |       setIsOpen={(next) => {

(react-hooks/refs)